waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -129,7 +129,9 @@ npmDepsForWasp spec =
             -- The web app only needs @prisma/client (we're using the server's
             -- CLI to generate what's necessary, check the description in
             -- https://github.com/wasp-lang/wasp/pull/962/ for details).
-            ("@prisma/client", show prismaVersion)
+            ("@prisma/client", show prismaVersion),
+            -- TODO(matija): should we import this conditionally, only if use use auth?
```

Comment:
I think that would be the best 👍 

We do that for PgBoss and e-mail sending deps for example.